topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -42,308 +51,526 @@ This tutorial explains how to qualify and calibrate the nitrate sensor mounted o
-> - 3g.  Click on “Build” on the bottom right of the pop-up
->
-> ![Image of how to build a collection from multiple netcdf files](../../images/bgc_calib/bgc_3a.png)
-> 
->
-> Once the collection is ready, everything is green in the history panel. It is possible to limit the history to useful files or collections by clicking on the eyes above the files list in the history.
+> <hands-on-title>Create a dataset collection (from non-empty history)</hands-on-title>
+> - Click on **Select item**
+> - Manually tick only the files of interest
+> - Click on **## of ## selected**
+> - Choose **Auto build list**
+> - In the pop-up window:
+>   - Set the name: `4903881`
+>   - Turn off **Remove file extension**
+>   - Click on **Build**
+>
+> ![Create collection - non-empty history](../../images/bgc_calib/collection_hist.png)
+> ![Collection build interface](../../images/bgc_calib/build_collection.png)
 {: .hands_on}
 
 
+> <hands-on-title>Change file extension (for WOA file)</hands-on-title>
+> - In the history panel, click on the **pencil icon** of the WOA NetCDF file
+> - In the central panel, go to the **Datatype** tab
+> - Set the **New type** to `NetCDF`
+> - Click on **Save**
+>
+> ![Change extension](../../images/bgc_calib/change_ext.png)
+{: .hands_on}
 
-{% include _includes/cyoa-choices.html option1="Tools" option2="Workflow" default="Tools" disambiguation="tutorialstyle"
-       text="Do you want to run the workflow or to discover the tools one by one ?" %}
 
 <div class="Workflow" markdown="1">
 
-> <hands-on-title> Run a predefined workflow </hands-on-title>
-> 4a. Click on “Workflows” on the vertical panel on the left. A pop-up is launched
-> 4b. Search “Argo-Glider Nitrate QCV” in the top search bar for run the workflow of one float
-> 
-> ![Image on how to get the right workflow](../../images/bgc_calib/bgc_workflow.png)
-> 
-> # Workflow human in the loop
-> 
-> Now you've got your workflow running, however this workflow is composed of both interactive and non interactive tools. Thus, for the interactive tools we need you !
-> 
-> > <tip-title>Copy pasting between computer and ODV</tip-title>
-> > You can expand the ODV left panel (where there are 3 dots, vertically) to access the "clipboard" menu and paste the content you want to paste on an ODV form. From there you can copy-paste everything from one side to the other. Then, click outside of this panel to collapse it.
-> >
-> > ![Image showing in transparent on the left of the ODV interface the clipboard](../../images/coastal_water_dyn/clipboard.png)
-> {: .tip}
-> 
-> > <tip-title>ODV - Disconnected</tip-title>
-> > If at one point your ODV interface becomes grey with a red panel on the top "X ODV - Disconnected", do NOT panic ;) you just need to reload your tab (circular arrow top left)
-> {: .tip}
+# Run a predefined workflow
+
+> <hands-on-title>Run a public workflow</hands-on-title>
+>
+> - Click on **Workflows** on the vertical panel on the left of the **Galaxy action list**  

```

Comment:
Not sure if we already have an FAQ for this, if so please use it, otherwise it would be great if you could make one? (I can also help)

Reply:
I'll try to do it and update the ones I need in the same time I guess

Reply:
OKay I tried something let me know if it's okay. If yes I will include those snippets in my tuto